homeassistant/components/automation/__init__.py

```diff
@@ -356,10 +356,11 @@ def _async_get_action(hass, config, name):
     script_obj = script.Script(hass, config, name)
 
     @asyncio.coroutine
-    def action(variables=None):
+    def action(variables=None, entity_id=None):
```

Comment:
I was just giving it the same behavior if the new parameter was not passed, but since the only caller is in this file I suppose it doesn't really matter.

Should both parameters be required, since they're both passed anyway?